interactors/nonceHandlerV3/addressNonceHandler.go

```diff
@@ -75,6 +79,18 @@ func (anh *addressNonceHandler) SendTransaction(ctx context.Context, tx *transac
 
 	select {
 	case response := <-ch:
+		if response.Error == nil {
```

Comment:
can be extracted in a new function, no need for the else branch

Reply:
done.